src/components/Dashboard/cells/sponsors.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+
+import Amadeus from '../assets/Sponsors/Amadeus.png';
```

Comment:
let's use lower_snake_case for image assets and folders
path should be ../assets/sponsors/amadeus.png